spec.go

```diff
@@ -12,9 +12,9 @@ var version = "(devel)"
 func Specification() sdk.Specification {
 	return sdk.Specification{
 		Name:        "weaviate",
-		Summary:     "<describe your connector>",
-		Description: "<describe your connector in detail>",
+		Summary:     "Conduit Connector for Weaviate",
+		Description: "Conduit Connector for Weaviate",
```

Comment:
also can we plz add a bit more details here, and more details in the `Destination` section from the readme
```suggestion
		Description: "Conduit Destination Connector for Weaviate",
```

Reply:
I added a little bit more. I'm not sure if I'm simply not creative for this, but I didn't find much to describe here, as the connector is rather simple. Mostly it's about the configuration.